Source/OCMock/OCClassMockObject.h

```diff
@@ -22,7 +22,7 @@
     Class               originalMetaClass;
 }
 
-- (id)initWithClass:(Class)aClass;
+- (id)initWithClass:(Class)aClass protocols:(NSArray *)protocols;
```

Comment:
This change is way too invasive. At an absolute minimum we'd have to keep the original `initWithClass:` method, and make it forward to the proposed new `initWithClass:protocols:` method. That way, the number of changes in the PR would be greatly reduced.